### PR TITLE
Updated activation_email.html to solve Issue #560

### PR DESCRIPTION
**Pull request by stefanZorcic**

### CODE REVIEW

**Comment by stefanZorcic** [comment]
Images of new email design, disregard clutter from {} elements
Screen Shot 2022-04-12 at 4 53 51 PM
Screen Shot 2022-04-12 at 4 53 56 PM

**Comment by stefanZorcic** [comment]
Updated image
Screen Shot 2022-04-12 at 5 12 31 PM
Screen Shot 2022-04-12 at 5 12 36 PM
s

**Comment by stefanZorcic** [comment]
Updated image
Screen Shot 2022-04-12 at 5 13 26 PM
Screen Shot 2022-04-12 at 5 13 31 PM
s

File: templates/registration/activation_email.html [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+        {{ activation_key }}
+        <br>
+        {% if SITE_ADMIN_EMAIL %}
+        <p>See you soon!</p>
+    </div>
+    <div style="margin-top:40px;background-color:#282828;padding: 40px;text-align:center;border-radius: 3px;">
+        <i style="color: white">If you have problems activating your account, feel free to send us an email at <a href="mailto:{{ SITE_ADMIN_EMAIL }}" style="color: white">{{ SITE_ADMIN_EMAIL }}</a>.</i>
+        <br>
+            {% else %}
+        <p style="color: white">See you soon!</p>
```
Comment (int-y1) [inline templates/registration/activation_email.html:29]: I don't see a reason to have `See you soon!` in both the `if` and `else` branches. Just put it before the if-statement.

Also, this white text doesn't work.
Capture

File: templates/registration/activation_email.html [hunk 2]
```diff
@@ -1,23 +1,33 @@
-<b>Thanks for registering on the {{ site.name }}! We're glad to have you.</b>
-<br><br>
-The last step is activating your account. Please activate your {{ SITE_NAME }} account in the next {{ expiration_days }} days.
-<br><br>
-Please click on the following link to activate your account:
-<p style="margin-left:1em">
-      <a href="http://{{ site.domain }}/accounts/activate/{{ activation_key }}/">http://{{ site.domain }}/accounts/activate/{{ activation_key }}</a>
-</p>
```
Comment (int-y1) [inline templates/registration/activation_email.html:1]: Remove this empty line.

File: templates/registration/activation_email.html [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
 
-Alternatively, you can reply to this message to activate your account.
-Your reply must keep the following text intact for this to work:
+<span style="display: block;text-align: center; justify-content: center;margin: 0 2px;border: 4px solid #FFDE05;border-radius: 6px;font-size: .95em;color: #444;margin-bottom:0.75em;margin-left:30%;margin-right:30%">
+    <div style="padding: 1em;">
+        <br>
+        <div>
+            <img style="display:table;margin-left: auto;margin-right: auto;"src="https://avatars1.githubusercontent.com/u/6934864?v=3&s=101" width="35%" height="30%">
```
Comment (int-y1) [inline templates/registration/activation_email.html:6]: `width="35%" height="30%"` creates a bad aspect ratio.

File: templates/registration/activation_email.html [hunk 4]
```diff
@@ -1,23 +1,33 @@
...
 
-<pre style="margin-left:1em">
-{{ activation_key }}
-</pre>
+        <h1 style="font-size:30px;display:inline;vertical-align: middle">Welcome to DMOJ!</h1>
+        <p>The last step is activating your account. Please activate your <b>{{ SITE_NAME }}</b> account in the next {{ expiration_days }} days.</p>
+        <br>
+        <p>Please click on the following link to activate your account:</p>
+        <p style="margin-left:1em">
+            <a href="http://{{ site.domain }}/accounts/activate/{{ activation_key }}/">http://{{ site.domain }}/accounts/activate/{{ activation_key }}</a>
```
Comment (int-y1) [inline templates/registration/activation_email.html:15]: `https` is more common nowadays.

File: templates/registration/activation_email.html [hunk 5]
```diff
@@ -1,23 +1,33 @@
...
+        </p>
+        <i>Alternatively, you can reply to this message to activate your account</i>
```
Comment (int-y1) [inline templates/registration/activation_email.html:17]: missing period

**Review by int-y1 — CHANGES_REQUESTED** [review]

File: templates/registration/activation_email.html (hunk 3 above)
Comment (int-y1) [reply templates/registration/activation_email.html:6]: Try using `max-width` and `max-height` in the `style`.

File: templates/registration/activation_email.html [hunk 6]
```diff
@@ -1,23 +1,33 @@
...
+        <br>
+        <div>
+            <img style="display:table;margin-left: auto;margin-right: auto;"src="https://avatars1.githubusercontent.com/u/6934864?v=3&s=101" width="35%" height="30%">
+        </div>
+        <br><br><br>
 
-<pre style="margin-left:1em">
-{{ activation_key }}
-</pre>
+        <h1 style="font-size:30px;display:inline;vertical-align: middle">Welcome to DMOJ!</h1>
```
Comment (int-y1) [inline templates/registration/activation_email.html:10]: Use `{{ SITE_NAME }}`, not `DMOJ`

**Comment by Xyene** [comment]
Superseded by https://github.com/DMOJ/online-judge/pull/2028.

**Pull request closed without merging**